doc/security/CVE-2022-0670.rst

```diff
@@ -0,0 +1,43 @@
+.. _CVE-2022-0670:
+
+CVE-2022-0670: Native-CephFS Manila Path-restriction bypass
+===========================================================
+
+Summary
+-------
+
+Users who were running OpenStack Manila to export native CephFS, who
+upgraded their Ceph cluster from Nautilus (or earlier) to a later
+major version, were vulnerable to an attack by malicious users. The
+vulnerability allowed users to obtain access to arbitrary portions of
+the CephFS filesystem hierarchy, instead of being properly restricted
+to their own subvolumes. The vulnerability is due to a bug in the
+"volumes" plugin in Ceph Manager. This plugin is responsible for
+managing Ceph File System subvolumes which are used by OpenStack
+Manila services as a way to provide shares to Manila users.
+
+Again, this vulnerability only impacts OpenStack Manila clusters which
```

Comment:
@zdover23 I think this _which_ should be a _that_.

Reply:
Both of these changes have been made.